apcd-cms/src/apps/admin_extension/views.py

```diff
@@ -129,100 +119,60 @@ def _set_extension(self, ext):
             'type': title_case(ext[5].replace('_', ' ')) if ext[5] else None,
             'requestor': title_case(ext[14]),
             'ext_outcome': title_case(ext[8]) if ext[8] else "None",
-            'ext_status': title_case(ext[7]),
+            'ext_status': title_case(ext[7]) if ext[7] else "None",
             'ext_id': ext[0],
             'submitter_id': ext[1],
             'approved_expiration_date': ext[4] if ext[4] else "None",
+            'current_expected_date': ext[2] if ext[2] else "None",
+            'requested_target_date': ext[3] if ext[3] else "None",
+            'applicable_data_period': _get_applicable_data_period(ext[6]) if ext[6] else "None",
+            'updated_at': ext[10],
+            'submitter_code': ext[11],
```

Comment:
may want to add checks to make sure these values exist here keeping in mind their expected type.

Reply:
added blank checks on most of the values (other than ext_id and submitter_id)